VARIABLES.md

```diff
@@ -702,15 +734,15 @@ Default:  "{{ monitoring_interceptors_enabled }}"
 
 ### kafka_connect_user
 
-Only use to customize Linux User Connect Service runs with. User must exist on host.
+Only use to customize Linux User Connect Service runs with. Default user is cp-kafka-connect
 
 Default:  "{{kafka_connect_default_user}}"
 
 ***
 
 ### kafka_connect_group
 
-Only use to customize Linux Group Connect Service user belongs to. Group must exist on host.
+Only use to customize Linux Group Connect Service user belongs to. Default group is confluent
```

Comment:
Modify this variable to customize the Linux Group that the Connect Service user belongs to. Default group is confluent.